docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML using the UI
+title: Configure SAML authentication using the Grafana user interface
+weight: 1150
+---
+
+# Configure SAML authentication using the Grafana user interface
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file. For instructions on how to set up SAML using the Grafana configuration file, refer to [Configure SAML authentication using the configuration file]({{< relref "../saml/" >}}).
+
+The Grafana SAML UI provides the following advantages over configuring SAML in the Grafana configuration file:
+
+- It is accessible by hosted Grafana users
+- It doesn't require Grafana to be restarted after a configuration update
+- Access to the SAML UI only requires access to authentication settings, so it can be used by users with limited access to Grafana's configuration
```

Comment:
By authentication settings do we refer to the new page? Since this is a new item in the menu, I am wondering if it would make sense to have additional docs update in [Configure authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/) to refer to the new admin page (which would include only SAML at the moment).

Reply:
Maybe we should mention that it's required RBAC configuration to grant access to non Grafana Admins?